decidim-core/config/locales/en.yml

```diff
@@ -1125,7 +1125,7 @@ en:
       newsletter:
         no_reply_notice: This email was sent from a notification email address that cannot accept incoming email. Please do not reply to this message.
         note: You received this email because you're subscribed to newsletters on %{organization_name}. You can change your settings on your <a href="%{link}">notifications page</a>.
```

Comment:
I think we need to also fix this link to the notifications page, if not I see some weirdness on the newsletter preview when clicking, as the URL opens inside the iframe: 


https://user-images.githubusercontent.com/717367/201637025-f6de6914-5dac-490a-aa1d-76b85685e4e7.mp4

(It should open outside or point to "#" as the others links)

Reply:
Done at b6f7c63cedbf34d280a5aeda3f17daf566f7caaa.